examples/chip-tool/commands/common/Commands.cpp

```diff
@@ -119,12 +119,13 @@ static void DetectAndLogMismatchedDoubleQuotes(int argc, char ** argv)
 
 } // namespace
 
-void Commands::Register(const char * clusterName, commands_list commandsList, const char * helpText)
+void Commands::Register(const char * commandSetName, commands_list commandsList, const char * helpText, bool isCluster)
 {
-    mClusters[clusterName].second = helpText;
+    mCommandSets[commandSetName].isCluster = isCluster;
+    mCommandSets[commandSetName].helpText  = helpText;
```

Comment:
Should you assert that 'helpText' can not be 'nullptr' if 'isCluster' is 'false' to enforce it ?